src/OrchardCore.Modules/OrchardCore.Settings/Drivers/DefaultSiteSettingsDisplayDriver.cs

```diff
@@ -49,7 +54,7 @@ public override Task<IDisplayResult> EditAsync(ISite site, BuildEditorContext co
 
         public override async Task<IDisplayResult> UpdateAsync(ISite site, UpdateEditorContext context)
         {
-            if (context.GroupId.Equals(GroupId, StringComparison.OrdinalIgnoreCase))
+            if (IsGeneralSettings(context))
```

Comment:
Can you please save dentition here by adding a return when not group? Look what I did in my PR.